go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/stretchr/testify v1.8.4
 	golang.org/x/net v0.23.0 // indirect
 	gomodules.xyz/jsonpatch/v2 v2.2.0
+	google.golang.org/protobuf v1.33.0 // indirect
```

Comment:
Shouldn't we suppose to bump direct reference that brings this indirect reference in?

Reply:
https://github.com/openshift/run-once-duration-override/pull/34